discordrb.json

```diff
@@ -57,6 +57,18 @@
             "voiceReceive": "no",
             "voiceSend": "yes",
             "voiceStateUpdate": "yes"
+        },
+        "misc": {
+            "supportsUserLogin": "yes",
+            "supportsBotLogin": "yes",
+            "cachesTokens": "yes",
```

Comment:
Not really, those should be covered by the other two sections - individual ones like joining invites already are. This is just for logging in